### PR TITLE
BUGFIX: SD-661 disable Name section on empty X axis of Scatter/Bubble chart

### PR DESCRIPTION
**Pull request by naibinh**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2519
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2333

# Related Jira tasks
- SD-661: https://jira.intgdc.com/browse/SD-661

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4963/ : SUCCESS in 7m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/303/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4964/ : SUCCESS in 5m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/304/ : SUCCESS in 7s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2613/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/305/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2015/ : SUCCESS in 13m 40s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1204.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/307/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2016/ : FAILURE in 1m 58s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4965/ : SUCCESS in 6m 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/306/ : SUCCESS in 6s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1204.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/308/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2017/ : SUCCESS in 23m 41s

File: src/internal/components/configurationPanels/BubbleChartConfigurationPanel.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default class BubbleChartConfigurationPanel extends ConfigurationPanelCon
                         pushData={pushData}
                     >
                         <NameSubsection
-                            disabled={controlsDisabled}
+                            disabled={controlsDisabled || itemsOnAxes.xaxis !== 1}
```
Comment (ptquang86) [inline src/internal/components/configurationPanels/BubbleChartConfigurationPanel.tsx]: please make a variable for this

File: src/internal/components/configurationPanels/ScatterPlotConfigurationPanel.tsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ export default class ScatterPlotConfigurationPanel extends ConfigurationPanelCon
                         pushData={pushData}
                     >
                         <NameSubsection
-                            disabled={controlsDisabled}
+                            disabled={controlsDisabled || itemsOnAxes.xaxis !== 1}
```
Comment (ptquang86) [inline src/internal/components/configurationPanels/ScatterPlotConfigurationPanel.tsx]: please make a variable for this

File: src/internal/components/configurationPanels/BubbleChartConfigurationPanel.tsx (hunk 1 above)
Comment (naibinh) [reply src/internal/components/configurationPanels/BubbleChartConfigurationPanel.tsx]: Fixed :)

File: src/internal/components/configurationPanels/ScatterPlotConfigurationPanel.tsx (hunk 2 above)
Comment (naibinh) [reply src/internal/components/configurationPanels/ScatterPlotConfigurationPanel.tsx]: Fixed :)

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4967/ : SUCCESS in 8m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/311/ : SUCCESS in 9s

**Comment by ptquang86** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4968/ : SUCCESS in 9m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/312/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2614/ : UNSTABLE in 10m 32s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4969/ : SUCCESS in 9m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1237/ : SUCCESS in 1m 07s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1631/ : SUCCESS in 18m 55s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2615/ : SUCCESS in 13m 06s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4970/ : SUCCESS in 5m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1238/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1633/ : SUCCESS in 7m 55s

**Pull request merged**